KtSPR-L1-T5-seconds-to-hh-mm-ss

DenisGubernatorov opened the pull request:
Вывод секунд в формате hh:mm:ss

DenisGubernatorov committed:
KtSPR-L1-T5-seconds-to-hh-mm-ss

Вывод секунд в формате hh:mm:ss

ievetrovdev commented on src/main/kotlin/lesson1/lesson1_task5.kt:
```diff
@@ -0,0 +1,9 @@
+fun main() {
+    val totalSeconds = 6480
+    val hours = totalSeconds / 3600
+    val minutes = (totalSeconds - hours * 3600) / 60
```
Чтобы снизить риск ошибок, предлагаю все “голые” числа вынести в константу. Кроме того это [магические числа](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)#%D0%9F%D0%BB%D0%BE%D1%85%D0%B0%D1%8F_%D0%BF%D1%80%D0%B0%D0%BA%D1%82%D0%B8%D0%BA%D0%B0_%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F), что тоже не является хорошей практикой.

ievetrovdev reviewed (changes requested):
Я оставил комментарий для доработки. чтобы правильно залить изменения, надо:
 - переключиться в Идее на нужную ветку
 - внести изменения и сделать коммит в эту ветку (следить чтобы в стейдж не добавились лишние задачи, которые не относятся к данной задаче)
 - отправить ссылку в бот на этот же пулл реквест к соответствующей задаче

DenisGubernatorov committed:
KtSPR--L1-T5-seconds-to-hh-mm-ss
Доработка кода согласно комментарию. Количество секунд в часе и минутах вынесены константы

DenisGubernatorov commented:
Доработки согласно комментарию

ievetrovdev reviewed (approved):
Отлично
